pkg/collect/host_run.go

```diff
@@ -0,0 +1,79 @@
+package collect
+
+import (
+	"bytes"
+	"encoding/json"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
+	troubleshootv1beta2 "github.com/replicatedhq/troubleshoot/pkg/apis/troubleshoot/v1beta2"
+)
+
+type HostRunInfo struct {
+	Command  string `json:"result"`
```

Comment:
Maybe change json to also be `"command"` since at first I wasn't sure if it was the command or the output, but see that it is the resulting command. Up to you though since it's just an aesthetic change.

Approving, but probably should wait to get @divolgin to give it a look.